repository-data/application/src/main/resources/hcm-actions.yaml

```diff
@@ -32,3 +32,9 @@ action-lists:
       /content/documents/administration/valuelists/lks/casestudyregions: reload
       /content/documents/administration/valuelists/lks/casestudysectors: reload
       /content/documents/administration/valuelists/lks/searchbankproviders: reload
+  - 1.0.24:
+      /content/documents/administration/valuelists/documenttypes: reload
+      /content/documents/administration/valuelists/listingpagetypes: reload
+      /content/documents/administration/valuelists/lks/newscategories: reload
```

Comment:
Since, the following are new value-lists, I think we do not need to reload them:
```
      /content/documents/administration/valuelists/lks/newscategories: reload
      /content/documents/administration/valuelists/dental/newscategories: reload
      /content/documents/administration/valuelists/digital-transformation/newscategories: reload
```

Reply:
Done.